sdks/java/extensions/sql/build.gradle

```diff
@@ -49,6 +50,7 @@ applyJavaNature(
     relocate "com.google.logging", getJavaRelocatedPath("com.google.logging")
     relocate "com.google.longrunning", getJavaRelocatedPath("com.google.longrunning")
     relocate "com.google.rpc", getJavaRelocatedPath("com.google.rpc")
+    relocate "com.google.api", getJavaRelocatedPath("com.google.api")
```

Comment:
I deleted this line because it breaks the SQL postcommit.

Honestly I am not sure how this line fix BEAM-8080. Could we have a minimal relocation(or a minimal exclude relocation) to both fix BEAM-8080 and pass the SQL postcommit?